contrib/podmanimage/stable/Containerfile

```diff
@@ -0,0 +1,56 @@
+# stable/Dockerfile
```

Comment:
```suggestion
# stable/Containerfile
```
I'm assuming this doesn't mess up quay.io

Reply:
Hmm, dunno why those comments are there.  Probably easier to just remove them.